cammac60/MacRae-Photo
# Photo comp

### What does this PR do? 

- Adds/styles the photo component. 
- Adds a new background image.
- Adds a landscape key to the data objects. 
- Adds the ability to reload an image page without the app crashing. 
- Adds several helper methods to App.
- Adds a loader directory and files for future use.

----------

### How to manually test: 

- `git fetch`
- `git checkout photo-comp`
- `npm test` to ensure all tests are passing. 
- 'npm start' => Go to the gallery and click on any image => Ensure that the image loads => Click back and forward buttons to ensure that you are taken to other images => Reload the page and make sure the image reloads => Go to the first image and click back to ensure that you're taken to the last image => From the last image click the forward button to ensure that you are taken to the first image. 

----------

### Tickets: 

- Resolves #22 
- Resolves #23 
- Resolves #38 
- Resolves #39 
- #30 has been moved to Blocked for now. This may need to be revisited when I get to #28 

----------

### Notes: 

- The styling of this component may need some major changes when #28 is addressed.
